### PR TITLE
Alpha

### PR DESCRIPTION
**Pull request by TD-D**

### CODE REVIEW

**Comment by Merp** [comment]
I take it those LC/FFS definitions are for proprietary stuff? If so I'd prefer they be in their own 'vendor branch' (ala MerpMod branch) with the appropriate base table structure. I can take care of this, but first you must have everything pushed as it will require a reset.

If you want to do it, there are a lot of ways to do this.. the first that comes to mind is to copy your master branch and delete the lc/ffs commits from this new branch. Then make the vendor tracking branch (based on the new master) and cherry pick the lc/ffs commits from your original master. After checking everything you can reset your original master to the new one.

**Comment by TD-D** [comment]
Hi Merrill - yes, it is - it's Vishnu Tuning's OFT roms - freely available,
but proprietary.  I've been helping them flesh out the definitions for the
BRZ, they are actually moving their customer onto Romraider, as it's a much
more comprehensive editor compared to their solution currently.

That's exactly what I'll do - I'll delete the LC related stuff, push it,
and then create a seperate branch with the definitions including the LC/FFS
stuff.  I'll give you a heads up when I've done it.

Excited to see your flex fuel stuff - looks interesting!  I'm going to move
to flex fuel when my 2.7 stroker is done, so am very keen.

You well otherwise?

Cheers,
Oren

**Comment by TD-D** [comment]
I've removed the LC/FFS xml from the 3 definitions concerned and repushed -
the commits as they stand now can be pulled in to your repository.

I've think rather than creating a whole new repository like Merpmod, to
create a seperate directory under Subarudefs for the OFT related
definitions.  This will only ever be definitions - not code (since it's a
commercial product).

Let me know how you want to proceed, and then I can put up the definition
with the LC/FFS seperately.

Cheers,
Oren

**Comment by Merp** [comment]
Doing well, hope you are too! I will have a look at these new roms soon :)

The OFT stuff belongs in a branch within SubaruDefs. This branch can then be kept up to date by rebasing as changes are brought into master. Sorry for the confusion, not an entirely separate repository. Although, they should have that too.. being "Open" and all :)

Also, to delete stuff, use:

```
 git rebase -i <commit hash before what you want to delete>
```

But first write down the hashes, then cherry pick them into the new branch. You'll need to force push master after this.

**Comment by TD-D** [comment]
Hey Merrill

I've created a new branch (OFT) and placed the three xml definitions in
there.

I don't think I will need to rebase, as I simply edited the proprietary
stuff out in the second last commit on the Alpha branch - unless you feel
it's necessary for me to completely kill the original commits?  If not,
then you should be able to pull it all in to your repository now.

Cheers,
Oren

**Comment by Merp** [comment]
I also just noticed you have a second 32bitbase under the BRZ directory. I don't know how this works if you open the repo in ECUFlash, but it isn't good practice. If the tables aren't ready to go into the 32bitbase, they really belong on a new branch. This way, pulling the experimental stuff into the alpha 32bitbase can be done using git instead of a text editor or clobbering a file, even if changes have been made to both bases! :)

**Comment by TD-D** [comment]
Yeah - agreed - it's also causing issue with Sharp Tune. It's been out for
a while now, I'm actually comfortable for it to replace the base
32bitbase.xml - no apparent issues.

**Comment by TD-D** [comment]
Hey Merrill

I’ve updated the Alpha 32bitbase.xml and removed the BRZ directory one, should be able to pull it all in now.

Cheers,

Oren

**Comment by TD-D** [comment]
Ok - hopefully this will do it?  I still need to create the seperate branch for the 32bitbase, but I have a full backup of my original branch.

**Comment by Merp** [comment]
Check out the network view, our Alpha branches are still diverged. Did you fetch before doing the rebase?

**Comment by TD-D** [comment]
Hmm.. I'm sure I did, I do it as a matter of habit as soon as I get into
Git Gui.

Let me try again when I'm home, fetching and rebase.

**Comment by TD-D** [comment]
OK, rebased again and force pushed onto origin/Alpha (I have kept a full
backup of the directory prior to rebasing, just in case).

Looks like that got the two together?

**Comment by TD-D** [comment]
If you can pull the rest in (other than 32bitbase.xml) I'll create a
seperate branch for an Alpha 32bitbase.xml
